### PR TITLE
Add the circleci config

### PR DESCRIPTION
**Pull request by bqbn**
Mostly borrowed from addons-frontend project.

Thanks for opening a Pull Request (PR), here are a few guidelines as to what we need in your PR before we review it.

Please delete anything that isn't relevant to your patch.

- [X] Add a description of the changes introduced in this PR.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+#
+# DOCKERHUB_REPO - docker hub repo, format: <username>/<repo>
+# DOCKER_USER - Login user for docker hub
+# DOCKER_PASS - Login password for docker hub user
+version: 2.0
+
+jobs:
+  release-tag: # build for the master branch and tags
+    machine: true
+    working_directory: ~/addons-frontend-code
```
Comment (willdurand) [inline .circleci/config.yml]: Is `addons-frontend-code` the name of the project we want to use? How about `addons-code-manager` instead?

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+          name: Build the container ( Next 3 steps )
+          command: IMG="image-$(date +%j).gz"; if [[ -e ~/addons-frontend-code/docker/$IMG ]]; then echo "Loading $IMG"; pigz -d -c ~/addons-frontend-code/docker/$IMG | docker load; fi
+      - run: >
+          printf '{"commit":"%s","version":"%s","source":"https://github.com/%s/%s","build":"%s"}\n'
+          "$CIRCLE_SHA1"
+          "$CIRCLE_TAG"
+          "$CIRCLE_PROJECT_USERNAME"
+          "$CIRCLE_PROJECT_REPONAME"
+          "$CIRCLE_BUILD_URL"
+          > version.json
```
Comment (willdurand) [inline .circleci/config.yml]: We'll have to update the `Dockerfile` to copy this `version.json` file.

I suggest we had a `version.json` file in this project (with `{}` as content) and the following in the `Dockerfile`:

```
# Add `version.json` file for ops
COPY --from=build /app/version.json ./
```

That way, we'll end up with a similar image locally or in docker hub (via circle-ci).

**Comment by bqbn** [comment]
@willdurand r? again?

**Comment by willdurand** [comment]
I am going to merge this patch because Circle-CI fails all builds :/

**Pull request merged**